### ISSUE TITLE
Add debugging option

### ISSUE THREAD

**Issue by stefancocora** [UKHomeOffice/kd#9]
To help with debugging , when running kd in debugging mode, cache locally in a `.kd/` directory the yaml or json response received from the kubelet.
@vaijab suggested that another option is to output the yaml/json from kubectl to stdout , that works too for debugging.

**Comment by stefancocora** [comment]
This kind of option would help with debugging this issue where kd gets this response from kubectl ( kd output below )

```
...
deployment "pttg-ip-api-testing" configured
Deployment failed. It has been superseded by another deployment.
```
There wasn't any other jenkins job running at the same time that would have been deploying a  _Deployment_

**Comment by vaijab** [comment]
Fixed by #14
